src/DataValues/DataValue.php

```diff
@@ -10,7 +15,7 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-interface DataValue extends \Hashable, \Comparable, \Serializable, \Immutable {
+interface DataValue extends Hashable, Comparable, Serializable, Immutable {
```

Comment:
Can you explain? I had the impression we agree that full qualified class names should be avoided.